introspection-engine/connectors/sql-introspection-connector/src/misc_helpers.rs

```diff
@@ -339,3 +339,14 @@ pub fn deduplicate_field_names(datamodel: &mut Datamodel) {
 pub fn columns_match(a_cols: &[String], b_cols: &[String]) -> bool {
     a_cols.len() == b_cols.len() && a_cols.iter().all(|a_col| b_cols.iter().any(|b_col| a_col == b_col))
 }
+
+pub fn replace_field_names(target: &mut Vec<String>, old_name: &str, new_name: &str) {
+    target
+        .iter_mut()
+        .map(|v| {
+            if v == old_name {
+                *v = new_name.to_string()
+            }
+        })
+        .count();
```

Comment:
I believe this call to `count` doesn't do anything here?

Reply:
It's needed to consume, but as discussed I'll use the more idiomatic `for_each(drop)`